pyccel/codegen/printing/fcode.py

```diff
@@ -452,28 +455,151 @@ def _print_Import(self, expr):
         return self._get_statement(code) + '\n'
 
     def _print_PythonPrint(self, expr):
+        end = '\n'
+        sep = ' '
+        code = ''
+        empty_end = FunctionCallArgument(LiteralString(''), 'end')
+        space_end = FunctionCallArgument(LiteralString(' '), 'end')
+        for f in expr.expr:
+            if f.has_keyword:
+                if f.keyword == 'sep':
+                    sep = str(f.value)
+                elif f.keyword == 'end':
+                    end = str(f.value)
+        args_format = []
         args = []
-        n = len(expr.expr)
-        for j, f in enumerate(expr.expr):
+        orig_args = [f for f in expr.expr if not f.has_keyword]
+
+        def formatted_args_to_print(fargs_format, fargs, end):
+
+            def split_long_argument(line):
+                line = line[1 : len(line) - 1]
+                length = len(line)
+                longest_chunk = length
+                cuts = []
+                positions = [idx for idx in range(length) if line[idx] == ' ']
+                if len(positions) > 12:
+                    positions = positions[int(len(positions) / 4) : 12]
+                for mask in range(1, 1<<len(positions)):
+                    mask_cuts = [positions[idx] for idx in range(len(positions)) if (1 << idx)&mask != 0 ]
```

Comment:
Codacy seems to have missed this, but in python it would be better to use enumerate here, i.e.:
```
mask_cuts = [pos for idx,pos in enumerate(positions) if (1 << idx)&mask != 0 ]
```

Reply:
Noted!